### ISSUE TITLE
Adding Data Tab out

### ISSUE THREAD

**Issue by Stoerfakt0r** [sqlitebrowser/sqlitebrowser#2815]
Is it possible to tab to the next Colum while adding new data, ignoring the suggestion? As of now pressing tab does nothing and I have to hit Enter to accept the suggestion:
![image](https://user-images.githubusercontent.com/62432294/131547243-5b0a431f-12fe-4d02-b1ed-3ccca7dc3dbc.png)

I have a lot of data to put into the DB manually and this gets annoying xD 
Also is there a shortcut for "new row"?  Maybe make it so a new row always get's offered (like SQL Server Management Studio has)

**Comment by chrisjlocke** [comment]
I tend to input via naked SQL sometimes a bit easier.  ```insert into tableA values (2,4,5,7),(4,5,6,7),(2,1,4,0)``` etc.
Depending on what data you're putting in, using statements can be fun... 😉😁

**Comment by Stoerfakt0r** [comment]
Yeah that's what I am doing now, but I also had some data that contained long texts I had to copy besides some Ids. That gets rather confusing after a while. I guess it would be a nice QoL feature :) But yeah in this case simple SQL is a rather fast option :)

**Comment by chrisjlocke** [comment]
Obviously the other solution is to use a normal spreadsheet, export to CSV, then import that.  Not all that pleasant...
Or, depending on your OS, you could use a third-party input tool (as SQLite is a damn easy database to program into).  Again, depends on what data you're putting in, how frequently, etc.

**Comment by Stoerfakt0r** [comment]
Actually I am generating C# code from a SQLite DB. I want to create a ton of Data in SQL, read it with EntityFramework and export a whole C# File with classes and data. I could use MS SQL and use SQL Server Management Studio (that's what I am used to), but decide to use SQLite, in case I will later use the file db instead of code.
The idea with CSV is a good workaround, didn't see the import option in DB Browser, nice :D 

But just adding the feature to press TAB to jump to the next column (skipping the suggestion popup) would mean I don't have to use another tool like Excel or even just Notepad. Breaks the workflow within this great tool (I really like it, superb!).

Guess I will stick with CSV then for now, thanks for the idea! :D

**Comment by mgrojo** [comment]
There is currently a preference for disabling the completion. In
Preferences > Data Browser >Threshold for completion, or similar (I speak
out of memory). Set it to 0 to disable it.

**Comment by Stoerfakt0r** [comment]
> There is currently a preference for disabling the completion. In Preferences > Data Browser >Threshold for completion, or similar (I speak out of memory). Set it to 0 to disable it.

This works nice. Now the only thing missing is a keyboard shortcut to create a new, empty row :P

**Comment by mgrojo** [comment]
Tab at the last cell of the table (last row, last column) will append a new row. I'm not discarding a new shortcut for a new row, only providing the current options.

**Comment by Stoerfakt0r** [comment]
> Tab at the last cell of the table (last row, last column) will append a new row. I'm not discarding a new shortcut for a new row, only providing the current options.

Wow, it's too easy. Thanks :D

**Comment by justinclift** [comment]
Does that mean this issue can be closed? :smile:
